resources/roadmap/index.md

```diff
@@ -239,9 +239,11 @@ meta_descr: meta_descr.roadmap
                                 <h3 class="months">{% t roadmap.comingsoon %}</h3>
                                     <li class="ongoing"><a href="https://github.com/Rucknium/OSPEAD">{% t roadmap.ospead %}</a></li>
                                     <li class="ongoing">{% t roadmap.bp++ %}</li>
-                                    <li class="ongoing">{% t roadmap.cuprate %}</li>
+                                    <li class="ongoing"><a href="https://cuprate.org/">{% t roadmap.cuprate %}</a></li>
+                                    <li class="ongoing"><a href="https://github.com/seraphis-migration/strategy/wiki/Our-Wallet-Rewrite-Project-After-Priority-Switched-From-Seraphis-to-FCMPs">{% t roadmap.fcmp-plus %}</a></li>
+                                    <li class="upcoming">BasicSwapDEX</li>
```

Comment:
BasicSwap is an independent project. Have we obtained consent from them to be present here as such?

Reply:
No